src/modules/core/components/InfoPopover/MemberInfoPopover.tsx

```diff
@@ -49,7 +49,11 @@ const MSG = defineMessages({
 
 const displayName = 'InfoPopover.MemberInfoPopover';
 
-const MemberInfoPopover = ({ colonyAddress, domainId, user }: Props) => {
+const MemberInfoPopover = ({
+  colonyAddress,
+  domainId,
+  user = { id: '', profile: { walletAddress: '' } },
```

Comment:
what is the reason behind those defaults? (if we check for existence below)

Reply:
That's needed to provide basic defaults, in case the `user` prop is not passed in, since just below this, we are destructuring the user object.

See: 
https://github.com/JoinColony/colonyDapp/blob/b4502176d21c36a3f7c5dae08e00ec269ce7b213/src/modules/core/components/InfoPopover/MemberInfoPopover.tsx#L58-L60

Even if the `user` value is set, it will still fail on the first component render, until all props have loaded their correct value.